http/logical.go

```diff
@@ -75,6 +75,30 @@ func buildLogicalRequest(core *vault.Core, w http.ResponseWriter, r *http.Reques
 		}
 	}
 
+	// If we are a read operation, try and parse any parameters
+	if op == logical.ReadOperation {
```

Comment:
It's valid to also use query params for POST. It might be good to parse this for POST as well, and then merge any form data with the query params, prioritizing form data.